solutions/quickstart/variables.tf

```diff
@@ -61,3 +61,9 @@ variable "topics" {
   description = "The list of topics to apply to resources. Only one topic is allowed for Lite plan instances. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/tree/main/solutions/quickstart/DA-schemas-topics-cbr.md)."
   default     = []
 }
+
+variable "service_credential_names" {
+  description = "The mapping of names and roles for service credentials that you want to create for the Event streams."
```

Comment:
This is going to need a markdown helper doc. Like this: https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/main/solutions/standard/DA-types.md

Reply:
@dishankkalra23 you need to link to the doc in the descrption